app/config/product_sources.json

```diff
@@ -0,0 +1,40 @@
+{
+  "focus_android": [
```

Comment:
Since you’re adding this one (it seems?), it should be documented in the accepted values in /documentation/